src/custom/estate/controllers/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```

Comment:
I can't comment the binary file. Binary files should not be committed. Just add ".pyc" in gitignore

Reply:
Noted.